src/psyclone/dynamo0p3.py

```diff
@@ -6274,14 +6509,14 @@ class HaloWriteAccess(HaloDepth):
     particular loop nest
 
     '''
-    def __init__(self, field):
+    def __init__(self, field, sym_table):
```

Comment:
docstring for the new argument please.

Reply:
Oops, done.

Reply:
Don't we add doc strings for `__init__` to the class description? Also could you please add a full stop for the first documentation line (for `param field`).

Reply:
Oops, oops and done.